xdem/coreg.py

```diff
@@ -892,15 +892,17 @@ class Deramp(Coreg):
     Estimates an n-D polynomial between the difference of two DEMs.
     """
 
-    def __init__(self, degree: int = 1, max_points: int = 5e5):
+    def __init__(self, degree: int = 1, subsample: Union[int, float] = 5e5):
         """
         Instantiate a deramping correction object.
 
         :param degree: The polynomial degree to estimate. degree=0 is a simple bias correction.
-        :param max_points: The maximum number of random points to extract for fitting (for speed-up).
+        :param subsample: Factor for subsampling the input raster for speed-up.
+        If < 1, will be considered a fraction of valid pixels to extract.
```

Comment:
What if it's 1.0? Maybe better to say "If <= 1, will be considered a fraction ..."

Reply:
Solved in 2ba3b5d